Index: config.m4
===================================================================
RCS file: /repository/pear/PECL/imagick/config.m4,v
retrieving revision 1.7
diff -u -r1.7 config.m4
--- config.m4	2 Dec 2002 21:25:21 -0000	1.7
+++ config.m4	10 Mar 2003 21:57:33 -0000
@@ -7,12 +7,12 @@
 [  --with-imagick[=DIR]       Include imagick support])
 
 if test "$PHP_IMAGICK" != "no"; then
-  if test -r $PHP_IMAGICK/include/magick/magick.h; then
+  if test -r $PHP_IMAGICK/include/GraphicsMagick/magick/magick.h; then
     IMAGEMAGICK_DIR=$PHP_IMAGICK
   else
     AC_MSG_CHECKING(for ImageMagick in default path)
     for i in /usr/local /usr; do
-      if test -r $i/include/magick/magick.h; then
+      if test -r $i/include/GraphicsMagick/magick/magick.h; then
         IMAGEMAGICK_DIR=$i
         AC_MSG_RESULT(found in $i)
       fi
@@ -36,32 +36,32 @@
     AC_MSG_ERROR(Please reinstall the ImageMagick distribution -
     magick.h should be in <imagemagick-dir>/include/magick/)
   fi
-  IMAGEMAGICK_CONFIG="Magick-config"
+  IMAGEMAGICK_CONFIG="GraphicsMagick-config"
   AC_MSG_CHECKING(for ImageMagick 5.4.5 or greater)
 
-  if ${IMAGEMAGICK_DIR}/bin/Magick-config --libs print > /dev/null 2>&1; then
-    IMAGEMAGICK_CONFIG=${IMAGEMAGICK_DIR}/bin/Magick-config
+  if ${IMAGEMAGICK_DIR}/bin/GraphicsMagick-config --libs print > /dev/null 2>&1; then
+    IMAGEMAGICK_CONFIG=${IMAGEMAGICK_DIR}/bin/GraphicsMagick-config
   else
-    if ${IMAGEMAGICK_DIR}/Magick-config --libs print > /dev/null 2>&1; then
-       IMAGEMAGICK_CONFIG=${IMAGEMAGICK_DIR}/Magick-config
+    if ${IMAGEMAGICK_DIR}/GraphicsMagick-config --libs print > /dev/null 2>&1; then
+       IMAGEMAGICK_CONFIG=${IMAGEMAGICK_DIR}/GraphicsMagick-config
     fi
   fi
   
   imagemagick_version_full=`$IMAGEMAGICK_CONFIG --version`
   imagemagick_version=`echo ${imagemagick_version_full} | awk 'BEGIN { FS = "."; } { printf "%d", ($1 * 1000 + $2) * 1000 + $3;}'`
   AC_MSG_RESULT($imagemagick_version_full)
-  if test "$imagemagick_version" -ge 5004005; then
+  if test "$imagemagick_version" -ge 1000000; then
     IMAGEMAGICK_LIBS=`$IMAGEMAGICK_CONFIG --libs`
     IMAGEMAGICK_LDFLAGS=`$IMAGEMAGICK_CONFIG --ldflags`    
   else
     AC_MSG_ERROR(ImageMagick version 5.4.5 or later is required to compile php with imagick support)
   fi
 
-  PHP_ADD_INCLUDE($IMAGEMAGICK_DIR/include)
+  PHP_ADD_INCLUDE($IMAGEMAGICK_DIR/include/GraphicsMagick)
   PHP_EVAL_LIBLINE($IMAGEMAGICK_LIBS, IMAGICK_SHARED_LIBADD)
   PHP_EVAL_LIBLINE($IMAGEMAGICK_LDFLAGS, IMAGICK_SHARED_LIBADD)
 
-  AC_CHECK_LIB(Magick,InitializeMagick, 
+  AC_CHECK_LIB(GraphicsMagick,InitializeMagick, 
   [ 
     AC_DEFINE(HAVE_IMAGICK,1,[ ])
   ],[
